kong/runloop/wasm.lua

```diff
@@ -541,18 +580,83 @@ local function get_filter_chain_for_request(route, service)
 end
 
 
+---@param filters kong.configuration.wasm_filter[]|nil
+local function discover_filter_metadata(filters)
```

Comment:
I think it's just about time to refactor some code out of `kong.runloop.wasm` and into a new `kong.wasm` module that is _not_ focused on runloop/proxy things, but that will be done later.